### PR TITLE
Remove hyperlink from labyrinth logo and add contribute option

### LINKED ISSUE #642: Labyrinth logo shouldn't be directly linked to the github repository
### Expected results
Clicking on Labyrinth logo on the main page index.html should not open github repository.

### Actual results
Link to repository can be given in the left menu bar rather than on the logo.

### Would you like to work on this
Yes

### PR DESCRIPTION
**Pull request by PradumanPannu**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [ ] Squashed the commits.

### Changes done in this Pull Request
Removed hyperlink from Labyrinth logo and added github repository link in left menu bar.

- http://rawgit.com/PradumanPannu/labyrinth/modify/index.html

- Fixes #642
   

### Screenshots,

contrbute

### CODE REVIEW

**Comment by duttaditya18** [comment]
What is up with so many people contribute from IIT Patna?

Firstly, welcome to this community. And then, can you give a little bit more explanation for this change?

Looks good to me nevertheless! :+1:

File: index.html [hunk 1]
```diff
@@ -57,10 +57,10 @@
             </div>
             <!-- When you add more languages do it in the above format-->
         </div>
-        <a href="https://github.com/fossasia/labyrinth">
+
```
Comment (niccokunzmann) [inline index.html]: you can remove the blank line.

File: index.html [hunk 2]
```diff
@@ -57,10 +57,10 @@
             </div>
             <!-- When you add more languages do it in the above format-->
         </div>
-        <a href="https://github.com/fossasia/labyrinth">
+          
           <h1 class="brand-logo center">Labyrinth</h1>
-        </a>
-        <ul class="right hide-on-med-and-down">
+        
+          <ul class="right hide-on-med-and-down">
```
Comment (niccokunzmann) [inline index.html]: I think, the indentation is a little off.
i.e. the ul could be two spaces to the left, so the li in it is indented.
Good indentation helps developing fast.
Maybe the h1 above it was wrong, I do not know.

**Comment by niccokunzmann** [comment]
@PradumanPannu Thanks for your contribution! I left some comments, so if you would like to help us increase the code quality, you can use these.

**Comment by PradumanPannu** [comment]
@duttaditya18 Thanks. The change is that the logo of labyrinth on the main page was used as a link to open its github page which was not a good way. Instead of this I added this link in the left menu bar as Contribute option.

File: index.html (hunk 2 above)
Comment (PradumanPannu) [reply index.html]: @niccokunzmann Thanks for the suggestion. I will correct it take care of it in future.

**Pull request merged**